ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/SignTransactionModal.qml

```diff
@@ -0,0 +1,190 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import QtQuick.Dialogs 1.3
+import "../../../../../imports"
+import "../../../../../shared"
+import "../../../../../shared/status"
+
+ModalPopup {
+    property var selectedAccount
+    property var selectedRecipient
+    property var selectedAsset
+    property var selectedAmount
+    property var selectedFiatAmount
+    property var selectedGasLimit
+    property var selectedGasPrice
+
+    id: root
+
+    //% "Send"
+    title: qsTrId("command-button-send")
+    height: 504
+
+    property MessageDialog sendingError: MessageDialog {
+        id: sendingError
+        title: qsTr("Error sending the transaction")
+        icon: StandardIcon.Critical
+        standardButtons: StandardButton.Ok
+    }
+    property MessageDialog sendingSuccess: MessageDialog {
+        id: sendingSuccess
+        //% "Success sending the transaction"
+        title: qsTrId("success-sending-the-transaction")
+        icon: StandardIcon.NoIcon
+        standardButtons: StandardButton.Ok
+        onAccepted: {
+            root.close()
+        }
+    }
+
+    onClosed: {
+        stack.reset()
+    }
+
+    function sendTransaction() {
+        let responseStr = walletModel.sendTransaction(root.selectedAccount.address,
+                                                 root.selectedRecipient.address,
+                                                 root.selectedAsset.address,
+                                                 root.selectedAmount,
+                                                 gasSelector.selectedGasLimit,
+                                                 gasSelector.selectedGasPrice,
+                                                 transactionSigner.enteredPassword)
+        let response = JSON.parse(responseStr)
+
+        if (response.error) {
+            if (response.error.includes("could not decrypt key with given password")){
+                transactionSigner.validationError = qsTr("Wrong password")
+                return
+            }
+            sendingError.text = response.error
+            return sendingError.open()
+        }
+
+        sendingSuccess.text = qsTr("Transaction sent to the blockchain. You can watch the progress on Etherscan: %2/%1").arg(response.result).arg(walletModel.etherscanLink)
+        sendingSuccess.open()
+    }
+
+    TransactionStackView {
+        id: stack
+        anchors.fill: parent
+        anchors.leftMargin: Style.current.padding
+        anchors.rightMargin: Style.current.padding
+        onGroupActivated: {
+            root.title = group.headerText
+            btnNext.label = group.footerText
+        }
+        TransactionFormGroup {
+            id: group1
+            headerText: qsTr("Send")
+            footerText: qsTr("Preview")
+
+            GasSelector {
```

Comment:
Yeah, that's the right design, but for now, I kept it simple by just using what is supported. We can later implement the right design in another PR, since the feature works anyway without it.

Reply:
OK, I created an issue for this so we don't forget: https://github.com/status-im/nim-status-client/issues/870.